### DISCUSSION TITLE
Credits beim Pförtner anzeigen

### DISCUSSION THREAD

**Discussion by nittka** [TVTower/TVTower#325] [Ideas]
Was haltet Ihr davon, die Credits aus der obersten Etage zum Pförtner zu verlegen? Im Moment ist bei ihm nichts Interessantes los. Den Raumplan dort nochmal anschauen zu können bringt auch wenig, wenn auf derselben Etage noch Fahrstuhl und der Planer sind. Zudem würde ein Raum freiwerden, in dem man ein leeres Studio unterbringen kann (direkt als zu vermieten).

Wenn der Pförtner selbst irgendwann Funktionalität bekommen soll, kann es ja eine "Credits"-Tafel in dem Raum geben, die man anklicken kann.

**Comment by jukey** [comment]
Ich finde die Idee gut, da ich so schon eher auf die Credits gestoßen wäre, als ich in 'Bettys Etage' bin.

Eine Frage die ich mir immer stelle ist: Wie sieht die Raumbelegung denn eigentlich normalerweise aus? Manchmal werden die Schilder so oft verschoben, dass ich nicht mehr genau weis, wo sich ein potenzieller 3er Studio verbergen könnte.

**Comment by GWRon** [comment]
Lokal hab ich den branch feat_porter schon angelegt - und gemerkt, es fehlt dafuer noch die Raumgrafik :)

Der Pfoertner selbst gibt (Dalli hat mir das letztens erst per Discord nochmal beschrieben) ja nur an, wenn Terroranschlaege "angekuendigt" sind.

Ich denke aber man kann dem auch noch per Dialog mitteilen, dass "Vandalen" schon wieder den Raumplan umgesetzt haben ... oder dass der Hausmeiser doch mal in einem der Bueros was aufkehren soll :) 

Und ja - die Credits koennten dann auch dort hin verschwinden.

Ein neuer Raum auf Hoehe von Betty nimmt aber deren luxorioese Alleinstellung ("Oberste Etage") weg.... muss man bedenken.

**Reply by nittka** [reply]
Man könnte den Kaffee-Automaten in die oberste Etage verschieben (Spielerbewegungen werden immer langsamer, wenn nicht hin und wieder ein Snack gekauft wird ;-) ). Schade ist die Platzverschwendung ja schon - wie wäre es denn mit einem besonders exklusiven und teuren Studio da oben...

**Comment by nittka** [comment]
Eine Idee für die oberste Etage wäre noch, ein allgemeines 3er-Studio, das man nicht mieten kann, anstelle der Credits. Gezahlt wird pro Stunde der Dreharbeiten (z.B. 12000 aufwärts pro Stunde). Ein Drehbuch darf man nicht stehenlassen (außer für die Produktion). Es kann immer nur eine Einzelproduktion gestartet werden (nicht mehrere Folgen auf einmal). Wenn eine Produktion abgeschlossen ist, darf man selbst die nächste frühestens am Folgetag starten.

Hintergrund dieser Idee ist, dass man so auch ohne Anmieten eines neuen Studios mal eine "bessere" Produktion starten kann (gegen ordentlichen Aufpreis). Und dass auch die KI die Möglichkeit hat, eine solche Produktion durchzuführen. Die Regeln für Einzelfolgen etc. sollen sicherstellen, dass alle Spieler eine Chance für die Nutzung bekommen.

**Comment by GWRon** [comment]
Statt dem Extrastudio (was dann auch "Druck" rausnimmt...) koennte auch ein Dienstleister existieren (beim Filmhaendler oder direkt Drehbuchhaendler) der fuer einen nicht unerheblichen Obolus die Produktion fuer einen vornimmt.

Gern koennte man auch fuer ein Drehbuch mehrere Produktionsoptionen anbieten (2-3 "fixe" Boxen die das Produktionsstudio und den Cast erwaehnen).
Der Aufpreis muss sich schon in der Zeitersparnis widerspiegeln.
Die Produktionsoptionen koennten auch den bereits im Hintergrund werkelnden Produktionsfirmen zugeordnet sein (und sich an deren XP usw orientieren).
